tableau/v_tableau.kipp_forward_survey.sql

```diff
@@ -0,0 +1,81 @@
+USE gabby
+GO
+
+CREATE OR ALTER VIEW tableau.kipp_forward_survey AS
+
+WITH alumni_data AS (
+SELECT 
+      c.salesforce_id_c
+     ,c.kipp_ms_graduate_c
+     ,c.kipp_hs_graduate_c
+     ,c.kipp_hs_class_c
+     ,c.college_match_display_gpa_c
+     ,c.kipp_region_name_c
+     ,c.[description]
+     ,c.gender_c
+     ,c.ethnicity_c
+
+     ,e.[name]
+     ,e.pursuing_degree_type_c
+     ,e.type_c
+     ,e.start_date_c
+     ,e.actual_end_date_c
+     ,e.major_c
+     ,e.status_c
+
+     ,s.survey_response_id
+
+     ,ROW_NUMBER() OVER (
+      PARTITION BY status_c,student_c
+      ORDER BY actual_end_date_c DESC) AS rn_latest
+
+FROM gabby.alumni.contact c
+JOIN gabby.alumni.enrollment_c e
+  ON c.salesforce_id_c = e.student_c
+JOIN gabby.surveygizmo.survey_detail s
```

Comment:
you can remove this JOIN from the CTE, you're already joining to the view below

Reply:
the output in survey_details is long (one row per question answer). I made it a CTE to match alumni data on just on the answer to one question (Salesforce ID), and so I could filter on rn_latest=1. Is there a way to do that on the main select? Basically I need every record in survey details to have a Salesforce ID tied to it.